src/ParallelAlgorithms/Amr/Actions/AdjustDomain.hpp

```diff
@@ -54,10 +54,13 @@ namespace amr::Actions {
 /// - Updates the Neighbors of the Element
 /// - Resets amr::Tags::Flag%s to amr::Flag::Undefined
 /// - Resets amr::Tags::NeighborFlags to an empty map
+/// - Mutates all return_tags of Metavariables::amr_mutators if the Mesh changed
 ///
-/// \warning At the moment, this action only updates the Mesh for a p-refined
-/// Element.  It does not update any data related to evolution or elliptic
-/// solves.
+/// \warning The current implementation assumes that:
+/// - any mutated items does not need access to the old Element.
```

Comment:
Not sure what "item" means in this context. Maybe use "mutator" or clarify the wording somehow?

Also please add information about the AMR mutators to the docs somewhere. Probably a small note here that refers to the AMR metavars docs that we talked about in the meeting could work. Also in the other actions that reach into the metavars.

Reply:
I've added a commit with protocols.  I'll use them in the actions and metavariables once you are happy with the protocols.